lib/components/Geojson.js

```diff
@@ -159,8 +159,10 @@ const Geojson = props => {
               image={props.image}
               pinColor={markerColor}
               zIndex={props.zIndex}
-              onPress={() => props.onPress(overlay)}
-            />
+              onPress={() => props.onPress && props.onPress(overlay)}
+            >
+              {props.markerChildren}
```

Comment:
Thoughts on using `component` for the prop name?

Reply:
Only `component` not referring to the `Marker` maybe `markerComponent` ?

Reply:
makerComponent sounds good